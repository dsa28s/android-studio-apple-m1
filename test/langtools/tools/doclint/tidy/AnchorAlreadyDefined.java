/*
 * @test /nodynamiccopyright/
 * @bug 8004832
 * @summary Add new doclint package
 * @library ..
 * @modules jdk.compiler/com.sun.tools.doclint
 * @build DocLintTester
 * @run main DocLintTester -ref AnchorAlreadyDefined.out AnchorAlreadyDefined.java
 */

// tidy: Warning: <.*> anchor ".*" already defined

/**
 * <a name="here">valid</a>
 * <a name="here">duplicate</a>
 * <h1 id="here">duplicate</h1>
 */
public class AnchorAlreadyDefined { }
