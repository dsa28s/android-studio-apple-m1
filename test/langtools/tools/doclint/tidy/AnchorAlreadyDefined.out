AnchorAlreadyDefined.java:15: error: anchor already defined: "here"
 * <a name="here">duplicate</a>
      ^
AnchorAlreadyDefined.java:16: error: anchor already defined: "here"
 * <h1 id="here">duplicate</h1>
       ^
2 errors
